### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Fixed the related projects section to reflect the Razor Compiler repo changes

### CODE REVIEW

**Review by TanayParikh — APPROVED** [review]

**Comment by TanayParikh** [comment]
Related; https://github.com/dotnet/aspnetcore/pull/44889

File: README.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ These are some other repos for related projects:
 * [Documentation](https://github.com/aspnet/Docs) - documentation sources for https://docs.microsoft.com/aspnet/core/
 * [Entity Framework Core](https://github.com/dotnet/efcore) - data access technology
 * [Runtime](https://github.com/dotnet/runtime) - cross-platform runtime for cloud, mobile, desktop, and IoT apps
-* [Razor Compiler](https://github.com/dotnet/razor-compiler) - the parser and the C# code generator for the Razor syntax
-* [Razor Tooling](https://github.com/dotnet/razor-tooling) - tools for working on Razor ASP.NET Core apps using [Visual Studio](https://visualstudio.com), [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/), and [Visual Studio Code](https://code.visualstudio.com/).
+* [Razor](https://github.com/dotnet/razor) - the parser and the C# code generator for the Razor syntax
```
Comment (danroth27) [inline README.md:41]: the Razor compiler and tooling for working with Razor syntax (.cshtml, .razor)

**Review by danroth27 — APPROVED** [review]
One small suggestion to also call out that the Razor repo contains the Razor tooling as well.

File: README.md (hunk 1 above)
Comment (mkArtakMSFT) [reply README.md:41]: Sounds good, thanks @danroth27 !

**Review by TanayParikh — APPROVED** [review]

**Pull request merged**